src/base/perf_monitor.h

```diff
@@ -634,8 +635,11 @@ typedef enum
   PSTAT_PB_AVOID_DEALLOC_CNT,
   PSTAT_PB_AVOID_VICTIM_CNT,
 
-  /* Scalability statistics */
-  PSTAT_SC_REPL_DELAY,
+  /* Scalability (and recovery) related statistics */
+  PSTAT_SCAL_REPL_DELAY,
+  PSTAT_SCAL_REPL_LOG_REDO_SYNC,
```

Comment:
I think a better naming here would be "Redo replication" instead of scalability. Replication is just one part of a bigger mechanism that provides the scalable property (albeit an important part).
```suggestion
  /* Redo recovery and replication statistics */
  PSTAT_REDO_REPL_DELAY,
  PSTAT_REDO_REPL_LOG_REDO_SYNC,
```

Reply:
Agreed. Done